### PR TITLE
combine-dv-keyshares-fixes: Small fixes

### PR DESCRIPTION
**Pull request by EridianAlpha**
## Summary
Small grammar and formatting fixes.

### CODE REVIEW

**Comment by OisinKyne** [comment]
Sorry to comment on a not directly related PR, but the quickstart guide to `charon combine`, can we put that in an 'Advanced Usage' section, or in the Charon CLI reference page for that command, or maybe in an FAQ section? And use a big: 

```
::::warning
Running a validator with a combined private key alongside the existing Distributed Validator Cluster you are combining will get you slashed. 
::::
```

I more or less want no one to ever need to use this page, definitely not as part of a quickstart. It is more in the case of a catastrophic failure of all charon nodes or the exit process such that combining is the only option.

**Comment by OisinKyne** [comment]
Also. CC @gsora lets get my requested change to the output folder structure in before we finalise this guide (and maybe merge it with your guide in the cli reference page, or one is the light detail and links to, the other more high detail page that eridian has written).

**Comment by gsora** [comment]
@OisinKyne waiting on your approval here: https://github.com/ObolNetwork/charon/issues/1836.

Code has already been reviewed, needs a quick 👌  on the output directory structure.

**Review by thomasheremans — APPROVED** [review]

**Pull request merged**
